symfony/apps/orangehrm/lib/form/admin/EmailConfigurationForm.php

```diff
@@ -48,7 +59,7 @@ public function configure() {
                             'sendmail' => 'Sendmail',
                             'smtp' => 'SMTP')
                         )),
-            'txtSendmailPath' => new sfWidgetFormInputText(),
+            'txtSendmailPath' => new sfWidgetFormInputText([], ['disabled' => 'disabled']),
```

Comment:
do we need to keep it as disabled input field? can we show it as a label.

Reply:
this is done.